applications/FluidDynamicsApplication/custom_processes/shock_capturing_physics_based_process.h

```diff
@@ -421,21 +402,6 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) ShockCapturingPhysicsBasedProcess :
         return rArray[0]*rArray[0] + rArray[1]*rArray[1] + rArray[2]*rArray[2];
     }
 
-    /**
-     * @brief Limiting function
-     * Function to limit a given value between two bounds
-     * @param s Value to limit
-     * @param s_0 Initial value
-     * @param s_max Maximum value
-     * @param s_min Minimum values
-     * @return double Provided value or the corresponding minimum or maximum one
-     */
-    double LimitingFunction(
-        const double s,
-        const double s_0,
-        const double s_max,
-        const double s_min = 0.0);
-
```

Comment:
Even though we're not using it, I wouldn't remove the non-smoothed limiting function.